backend/src/main/java/com/example/backend/controller/TaskController.java

```diff
@@ -9,6 +9,7 @@
 import java.util.List;
 
 @RestController
+@CrossOrigin
```

Comment:
Ce face CrossOrigin si dc e pusa in clasa asta?

Reply:
am pus la toate controllerele ca sa pot face request-uri de pe device-uri diferite fara sa imi iau eroare cors